modules/rosa-attaching-the-policy.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-attaching-the-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-attaching-the-policy_{context}"]
+= Attaching the Identity-based IAM policy
+
+:context: rosa-attaching-the-policy
+
+toc::[]
+
+Once you have created an Identity-based IAM policy, you need to attach it to the relevant IAM users, groups, or roles in your AWS account. This enforces IP-based role assumption prevention for those entities.
+
+.Procedure
+
+. Navigate to the IAM console in the AWS Management Console.
+. Select the IAM user, group, or role to which you want to attach the policy.
```

Comment:
From my perspective we need to be more specific as to which user, group, or role should be selected.

Reply:
Can SME or @yuwang-RH help here? Thank you!

Reply:
Considering the urgency of this Jira, would really appreciate if we get detailed information or comments if any for this PR. This will help us to incorporate the changes (if any) asap. Thank you!

Reply:
@geowa4 Are any updates needed here? Thank you for your support!

Reply:
Please see slack thread: https://redhat-internal.slack.com/archives/CCX9DB894/p1695974964400189?thread_ts=1695913650.974079&cid=CCX9DB894